modules/server/src/main/resources/db/migration/data/V100__create_protocols_table.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE protocols (
```

Comment:
Is there any reason to start counting on `V100` ?

Reply:
No specific reason, I thought that maybe it would be reasonable to add more migration scripts before this, and then forgot to rename the file. I'm changing this.